docs/release_notes.rst

```diff
@@ -19,6 +19,65 @@ Release Notes
 
 
 .. ddtrace-release-notes::
+    "1.1.0":
+      ignore_notes:
+        # Ignore 1.0 release notes
+        - "fix-pyston-memalloc-e8f4002e5c11df8c.yaml"
+        - "log-correlation-context-0ca54b5caf687b2d.yaml"
```

Comment:
this one probably shouldn't be ignored. It was backported to 1.0 in 1.0.1 so it would still be worth including for users upgrading from 1.0.0 or below.

Reply:
```suggestion
```